addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4416,6 +4447,10 @@ export class OdooEditor extends EventTarget {
     }
 
     _onMousemove(ev) {
+        this.options.pointerInsideCell = true;
+        if (this._columnUi.classList.contains('d-none')) {
+            this._columnUi.classList.remove('d-none');
```

Comment:
@visp-odoo hello can you explain me the purpose of this, for the record why are we doing this?

Reply:
@shsa-odoo I have written this code for solving the bug mentioned by SHRA in review of 23 Aug 2023.

In this bug i face problem to `hide` it using `visiblity` because when we have hoverOnTdBorder then the visiblity of `tableUiMenu` is set to `visible !important` in scss.So, i can not set its `visiblity: hidden`.For that reason i have hide this using `d-none`.